test/suites/logscan/log-scan.robot

```diff
@@ -40,21 +40,25 @@ Should Find MicroShift Is Ready
     [Tags]    robot:exclude
     Pattern Should Appear In Log Output    ${CURSOR}    MICROSHIFT READY
 
-
 *** Keywords ***
 Setup
     [Documentation]    Test suite setup
     Check Required Env Variables
     Login MicroShift Host
-    Setup Kubeconfig    # for readiness checks
+    Stop MicroShift
+    Cleanup MicroShift    --all    --keep-images
     # Save the journal cursor then restart MicroShift so we capture
     # the shutdown messages and startup messages.
```

Comment:
This comment could do with updating now that the logic of Setup is changing. Explaining why we stop the service, do a full clean up, then do the cursor stuff and start & stop it would be good.